src/projects/atlantic-wave-sdx.js

```diff
@@ -75,10 +82,14 @@ export const AtlanticWaveSdx = () => {
         <img src={ atlanticWavePromo } alt="Atlantic Wave-SDX Live Demo Promo" />
       </Link>
 
+      <Link to="https://drive.google.com/file/d/1UhgoGfSOsACiM3lTdZXxRQ2_yhOCzYR4/view?usp=sharing" className="center">
```

Comment:
it would be simpler to use a block (as opposed to inline) text container that controls text alignment here instead of extending/adding complexity to the inline `Link` component.

something like this gets to the same result with existing functionality and doesn't introduce complexity:

```
<Paragraph align="center">
  <Link to="https://drive.google.com/file/d/1UhgoGfSOsACiM3lTdZXxRQ2_yhOCzYR4/view?usp=sharing">
    <img src={ amLightLogo } alt="AmLight Logo" width={350}/>
  </Link>
</Paragraph>
```